bhclub.html

```diff
@@ -104,6 +104,15 @@ <h1>About</h1>
         </section>
         <section id="events">
             <h1>Events</h1>
+            <div>
+                <img src="./assets/bhclub/events.svg" alt="Illustration of a computer monitor with Ricky the Brick on it. Ricky is smiling, winking, and holding his arms up in a cheer.">
+                <div>
+                    <h2>BrickHack hackathon</h2>
+                    <p>Held each Feburary</p>
+                    <p>BrickHack is a 24-hour hackathon events that attracks 600+ hackers from around the world each year. Hackers work in teams to compete in different prize-awarded categories while learning new skills along the way. To learn more about this hackathon, check out the event's website!</p>
+                    <a class="btn-dark" href="index.html">VISIT BRICKHACK EVENT SITE</a>
```

Comment:
Thoughts on changing to `Visit event site`?

Reply:
Can do